packages/@lwc/rollup-plugin/src/index.js

```diff
@@ -45,7 +56,18 @@ module.exports = function rollupLwcCompiler(pluginOptions = {}) {
             customResolvedModules = [...userModules, ...DEFAULT_MODULES, { dir: customRootDir }];
         },
 
-        resolveId(importee, importer) {
+        async resolveId(importee, importer) {
+            const { filename, query, scopeToken } = parseQueryParamsForScopeToken(importee);
```

Comment:
I don't think there is anything needed to do in the `resolveId`. As far as I understand the code here, the `importee` is already a resolved scoped CSS file (with a query string). The logic in `load` should be sufficient.

Reply:
Hm, you're right. Can't recall why I thought I needed to modify `resolveId`.

Reply:
`resolveId` part is gone now.